recipe-client-addon/lib/NormandyApi.jsm

```diff
@@ -62,7 +62,10 @@ this.NormandyApi = {
   },
 
   async getApiUrl(name) {
-    const apiBase = prefs.getCharPref("api_url");
+    let apiBase = prefs.getCharPref("api_url");
+    if (!apiBase.endsWith('/')) {
+      apiBase += '/';
```

Comment:
If you use a URL constructor here to parse the URL, you get 2 bonuses over this approach:
1) it'll be correct in the face of querystrings / hashes (the latter of which I imagine we can just drop anyway?) without changing those.
2) it'll normalize things using our 'regular' URL parser and also allow us to formally catch invalid URLs before passing them to fetch (and throw an appropriate error).

However, that would be slightly more code. I'm fine with either approach if we're comfortable with this butchering query strings / hashes if any occur (and still having the 301 in that case).